app/grandchallenge/workstation_configs/models.py

```diff
@@ -264,15 +264,81 @@ def save(self, *args, **kwargs):
 
 class WindowPreset(TitleSlugDescriptionModel):
     width = models.PositiveIntegerField(
-        validators=[MinValueValidator(limit_value=1)]
+        blank=True, null=True, validators=[MinValueValidator(limit_value=1)]
     )
-    center = models.IntegerField()
+    center = models.IntegerField(blank=True, null=True)
+
+    lower_percentile = models.PositiveSmallIntegerField(
+        blank=True, null=True, validators=[MaxValueValidator(limit_value=100)]
+    )
+
+    upper_percentile = models.PositiveSmallIntegerField(
+        blank=True, null=True, validators=[MaxValueValidator(limit_value=100)]
+    )
+
+    def _validate_percentile(self):
+        if self.upper_percentile < self.lower_percentile:
+            raise ValidationError(
+                f"Upper percentile ({self.upper_percentile}%) should be below the "
+                f"lower percentile ({self.lower_percentile}%)"
+            )
+
+    def _validate_fixed(self):
+        pass
+
+    def clean(self):
```

Comment:
This is missing a call to `super().clean()`.

Reply:
I'll add it, but what is the reasoning behind it? Currently, there is no super to call upon. Is this to make it future proof for when something is added upstream of the inheritance?

Reply:
> Currently, there is no super to call upon.

There is.

Reply:
Apart from the `pass` base class one?

Reply:
How do you know that it will always return pass? How about in the next major revision? Doesn't matter what it returns, it is there, call it.

Reply:
So what I surmise it's about the future proving of call. Would it then also make sense to pass the cleaned data? As in:
```
def clean(self):
  cleaned_data = super.clean()
  [...]
  return cleaned_data
```

Reply:
No, models do not have a concept of cleaned data (forms do, which is where you would do this). The requirement of this method is only to raise a `ValidationError` if the set of values are not valid which is then handled by `full_clean`.